.vib/ruby/goss/ruby.yaml

```diff
@@ -0,0 +1,28 @@
+command:
+  check-installed-gems:
+    exec: gem list --local
+    exit-status: 0
+    stdout:
+      {{ range $module := .Vars.modules }}
+      - {{ $module }}
+      {{ end }}
+  check-ssl:
+    exec: ruby -rnet/https -e "Net::HTTP.get URI('https://bitnami.com')"
+    exit-status: 0
+  {{ range $flag := .Vars.flags }}
+  check-{{ $flag }}:
+    exit-status: 0
+    exec: grep -r {{ $flag }} /opt/bitnami/ruby/include/
+    stdout:
+      - config.h
+  {{ end }}
+  check-bundle:
+    exec: cd /tmp && echo "source 'https://rubygems.org'" > Gemfile && bundle
+    timeout: 30000
```

Comment:
There is no need to use `timeout` here. The execution for `jruby` was slower, but here it seems to be faster